### DISCUSSION TITLE
usernames

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#239] [General]
Hello, what are your bark usernames?

**Comment by dumorando** [comment]
mine is dumo

you can see all the users herer https://bark.dumorando.com/users

**Reply by mariocraft987** [reply]
> mine is dumo
> 
> you can see all the users herer https://bark.dumorando.com/users

All I see is "there are 8 users on Bark!", nvm its just the snail theme that's making it impossible to see

**Comment by Rubiidev-IPad** [comment]
make rubii484bark part of the bark team

**Comment by nmsderp** [comment]
ahem https://bark.dumorando.com/profile/nmsderp

**Comment by raynepaws** [comment]
[breakfast](https://bark.dumorando.com/profile/breakfast) :)
